packages/expo-analytics-segment/src/Segment.ts

```diff
@@ -0,0 +1,106 @@
+import { Platform } from 'react-native';
+import { UnavailabilityError } from 'expo-errors';
+
+import ExponentSegment from './ExponentSegment';
+
+export function initialize(options: { androidWriteKey?: string; iosWriteKey?: string }): void {
```

Comment:
In other places we pull out options into another type -> adding more options doesn't cause the method signature to get super tall, and we can export the options type if needed:

```suggestion
type SegmentOptions = {
  androidWriteKey?: string;
  iosWriteKey?: string;
};
```